lib/saml/user_attributes/ssoe.rb

```diff
@@ -11,6 +11,7 @@ class SSOe
                                    uuid idme_uuid sec_id mhv_icn mhv_correlation_id mhv_account_type
                                    dslogon_edipi loa sign_in multifactor].freeze
       IDME_GCID_REGEX = /^(?<idme>\w+)\^PN\^200VIDM\^USDVA\^A$/.freeze
+      INBOUND_AUTHN_CONTEXT = 'urn:oasis:names:tc:SAML:2.0:ac:classes:Password'
```

Comment:
should this have a `.freeze` appended?

Reply:
good idea, ill add it

Reply:
actually thats giving me a linting error.  @kfrz are ruby constant strings by default frozen?

`lib/saml/user_attributes/ssoe.rb:14:31: C: Style/RedundantFreeze: Do not freeze immutable objects, as freezing them has no effect.`